rmgpy/tools/canteramodel.py

```diff
@@ -63,18 +64,45 @@ class CanteraCondition(object):
 
     """
 
-    def __init__(self, reactor_type, reaction_time, mol_frac, T0=None, P0=None, V0=None):
+    def __init__(self, reactor_type, reaction_time, mol_frac, surface_mol_frac=None, T0=None, P0=None, V0=None):
         self.reactor_type = reactor_type
         self.reaction_time = Quantity(reaction_time)
 
         # Normalize initialMolFrac if not already done:
         if sum(mol_frac.values()) != 1.00:
+            logging.warning('Initial mole fractions do not sum to one; normalizing.')
+            logging.info('')
+            logging.info('Original composition:')
+            for spec, molfrac in mol_frac.items():
+                logging.info('{0} = {1}'.format(spec, molfrac))
             total = sum(mol_frac.values())
+            logging.info('')
+            logging.info('Normalized mole fractions:')
             for species, value in mol_frac.items():
                 mol_frac[species] = value / total
+                logging.info('{0} = {1}'.format(spec, mol_frac[species]))
+            logging.info('')
 
         self.mol_frac = mol_frac
 
+        if surface_mol_frac:
+            # normalize surface mol fractions
+            if sum(surface_mol_frac.values()) != 1.00:
+                logging.warning('Initial surface mole fractions do not sum to one; normalizing.')
+                logging.info('')
+                logging.info('Original composition:')
+                for spec, molfrac in surface_mol_frac.items():
+                    logging.info('{0} = {1}'.format(spec, molfrac))
```

Comment:
This is a nitpick but can we use f-strings instead of format
```suggestion
                    logging.info(f'{spec} = {molfrac}')
```